Resources/public/Admin.js

```diff
@@ -103,6 +103,31 @@ var Admin = {
                 checkboxClass: 'icheckbox_square-blue',
                 radioClass: 'iradio_square-blue'
             });
+
+            var previousIndex;
+
+            jQuery('tbody .iCheck-helper', subject).click(function (evt) {
```

Comment:
I updated the PR, what do you think ? I also improve the code to handle range selection when iCheck is disabled !